client/base_client.go

```diff
@@ -6,6 +6,7 @@ import (
 )
 
 type BaseClient interface {
+	GetAccountSid() string
```

Comment:
I don't think this needs to be part of the interface.

Reply:
The client [here](https://github.com/twilio/twilio-go/blob/subaccount-support/twilio/rest/api/v2010/api_default.go#L26) is of the type `BaseClient` and the only way I could expose the AccountSid in the implementation of that baseclient was to have a getter in the interface.